Arena.toml

```diff
@@ -1115,787 +1115,82 @@ document = "getwilds/ww-vc-trio:/ww-vc-trio.wdl"
 message = "ww-vc-trio.wdl:98:29: note[CallInputSpacing]: call input not properly spaced"
 permalink = "https://github.com/getwilds/ww-vc-trio/blob/c2c13e85efda8ac7aca7f8765fbaa7c2cacb08b2/ww-vc-trio.wdl/#L98"
 
-[[diagnostics]]
```

Comment:
All of these are because the workflows repo updated and @a-frantz resolved most of the lints.